pkg/controller/constrainttemplate/stats_reporter_test.go

```diff
@@ -60,11 +65,11 @@ func TestReportIngestion(t *testing.T) {
 			t.Errorf("ingestDuration tags does not match for %v", tag.Key.Name())
 		}
 	}
-	if durationValue.Min != expectedDurationMin {
-		t.Errorf("Metric: %v - Expected %v, got %v. ", ingestDuration, durationValue.Min, expectedDurationMin)
+	if durationValue.Min != wantMinIngestDurationSeconds {
```

Comment:
nit: might be nice to have `durationValue` include the word "got", since we've now got half of the want / got convention

Reply:
Good point; will do